static/skywire-manager-src/src/assets/i18n/de.json

```diff
@@ -330,26 +471,43 @@
     },
     "dialog": {
       "remote-key": "Remote öffentlicher Schlüssel:",
+      "label": "Bezeichnung (optional)",
       "transport-type": "Transport-Typ",
       "success": "Transport erstellt.",
+      "success-without-label": "Der Transport wurde erstellt, aber die Bezeichnung konnte nicht gespeichert werden.",
       "errors": {
         "remote-key-length-error": "Der remote öffentliche Schlüssel muss 66 Zeichen lang sein.",
         "remote-key-chars-error": "Der remote öffentliche Schlüssel darf nur hexadezimale Zeichen enthalten.",
         "transport-type-error": "Ein Transport-Typ wird benötigt."
       }
+    },
+    "filter-dialog": {
+      "online": "Der Transport muss sein",
+      "id": "Die ID muss enthalten",
+      "remote-node": "Der Remote Schlüssel muss enthalten",
+
+      "online-options": {
+        "any": "Online oder offline",
+        "online": "Online",
+        "offline": "Offline"
+      }
     }
   },
 
   "routes": {
     "title": "Routen",
+    "info": "Pfade um Remote Visor, zu denen ein Transport aufgebaut wurde zu erreichen. Routen werden bei Bedarf automatisch generiert.",
```

Comment:
`Netzwerkpfade zum erreichen von Remote Visor.` may be a bit more concise and illuminating.

Reply:
Yes, updated